media-tv/plex-media-server/plex-media-server-1.23.1.4571.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd unpacker
+
+DESCRIPTION="Free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+
+MY_PV="${PV}-6119e8eed"
+
+MY_URI="https://downloads.plex.tv/plex-media-server-new"
```

Comment:
Not a hard requirement but it would be nice to re-arrange these to keep the usual variable block together.

Not everyone does the same but usually I'd do something like:
```
inherit ...

MY_PV=
MY_URI=

DESCRIPTION=
HOMEPAGE=
SRC_URI=
S=

LICENSE=
```